app/mailers/received_kudos_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class ReceivedKudosMailer < ApplicationMailer
+
+  # Generated using `docker-compose run web rails generate mailer ReceivedKudosMailer`
+  # NOTE: associated views need to be created in ../views/received_kudos_mailer/
+
+  def received_kudos_notification(kudo)
+    @kudo = kudo
+    @giver = User.find_by(id: @kudo.giver_id)
+    @receiver = User.find_by(id: @kudo.receiver_id)
+    @url = 'http://kudos.factual.com'
```

Comment:
This URL should probably be passed in with the docker config. Since we don't have deep links to individual kudos this will probably be a somewhat jarring experience but that's probably not something to fix in this PR.